.gitignore

```diff
@@ -28,6 +28,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 yarn.lock
+package-lock.json
```

Comment:
What's the reasoning behind ignoring the lock files?

Reply:
There were some issues previously due to both yarn-lock and package-lock being committed. Maybe we should choose one to prevent issues and commit only the lock file for that to keep it consistent?

Reply:
Yeah there definitely can be confusion when both are included as adding new dependencies through npm won't update yarn's lock and vice versa. I'd strongly recommend making sure there is one package lock file and would also strongly recommend making the project yarn only or npm only.

That might be worth making an issue for which I'll go do now.

Reply:
Made an [issue](https://github.com/yisselda/SlackableThemes/issues/32) for the above!